consensus/src/persistent_liveness_storage.rs

```diff
@@ -87,6 +88,7 @@ impl LedgerRecoveryData {
         &self,
         blocks: &mut Vec<Block>,
         quorum_certs: &mut Vec<QuorumCert>,
+        highest_ordered_cert: Option<WrappedLedgerInfo>,
```

Comment:
this needs to be persisted, can't be passed in

Reply:
Done.